src/GraphQL.MemoryCache/GraphQLBuilderExtensions.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using GraphQL.DI;
+
+namespace GraphQL.Caching
+{
+    /// <inheritdoc cref="GraphQL.GraphQLBuilderExtensions"/>
+    public static class GraphQLBuilderExtensions
```

Comment:
I would prefix each class for extensions:
`GraphQLBuilderExtensions` -> `MemoryCacheGraphQLBuilderExtensions`
MS does it, I saw such naming a lot.

Reply:
If we move the methods’ namespaces, then we should for sure. If not then…..

Reply:
For v5, this will not be critical changes.

Reply:
#2790